readme/skills.md

```diff
@@ -0,0 +1,126 @@
+# Minimum Skillset for Using BLT
+BLT is a tool that reaches far beyond Drupal development, and because of this developers planning on using it should working knowledge of the following technologies in order to be successful.
+
+See the [System Requirements](../INSTALL.md) for installing / using BLT. It is strongly urged that you use a Mac for local development, although certain versions of Linux and Windows 10 can also be used. Other operating systems ***should not*** be used for BLT development.
+ 
+## Enterprise Web Development
+### Virtual Environment
+#### [DrupalVM](https://www.drupalvm.com/)
+This project aims to make spinning up a simple local Drupal test/development environment incredibly quick and easy, and to introduce new developers to the wonderful world of Drupal development on local virtual machines (instead of crufty old MAMP/WAMP-based development).
+
+**Recommended Reading:**
+* [Drupal VM Quickstart Guide](https://github.com/geerlingguy/drupal-vm#quick-start-guide)
+* [BLT Onboarding](onboarding.md/#initial-setup)
+
+### Dependency Management
+####[Composer](https://getcomposer.org/): 
+Composer is a tool for dependency management in PHP. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you.
+
+**Recommended Reading:**
+ * [Getting Started with Composer](https://getcomposer.org/doc/00-intro.md)
+ * [BLT Dependency Management](dependency-management.md)
+
+**Common Commands:**
+ * [composer install](https://getcomposer.org/doc/03-cli.md#install)
+ * [composer update](https://getcomposer.org/doc/03-cli.md#update)
+ * [composer require](https://getcomposer.org/doc/03-cli.md#require)
+ 
+####[NPM](https://www.npmjs.com/)
+npm is the package manager for JavaScript. Find, share, and reuse packages of code from hundreds of thousands of developers — and assemble them in powerful new ways.
```

Comment:
This is not part of BLT. It is an aspect of Cog. We need to indicate that this isn't required. Perhaps a separate "optional" section.